PostingGuidelines.md

```diff
@@ -0,0 +1,28 @@
+What Should Be Posted
+=====================
+* **DO** post items that would be interesting or useful to the technical / programming community
+* **DO** post original blog posts where you have developed a solution to a problem
+* **DO** post summaries of other blog posts and how they apply to our work - make sure to link to to the source for these
+* **DO** post personal or project achievements if it relates to a particular project or initiative
+* **DO** post links to interesting technologies and articles and explain how we might use them. For these be sure to tag them with "InterestingTech" - Tags coming soon
+* **DO NOT** post internal documentation even if it does not contain sensitive information
+
+
+How To Post
+===========
+* Checkout the latest version of the [repository](https://github.com/emory-libraries/emory-libraries.github.io)
+* Create a new branch for your post
+* Make sure you have an entry for your username in `_data/_authors.yml`
+* Create a file in the `_posts` directory of the format `YYYY-MM-DD-TITLE.md`
```

Comment:
@athom09 I added some notes to the readme about using octopress to create drafts and publish them, which should handle the basic naming and layout settings for us.  Do we want to make that the recommendation?  We might want to move some of the post-specific instructions from the readme into this document.